Makefile

```diff
@@ -27,32 +26,9 @@ cover: $(COVER_FILE) ## Output coverage in human readable form in html
 	go tool cover -html=$(COVER_FILE)
 	rm -f $(COVER_FILE)
 
-
 .PHONY: lint
 lint: tools ## Check the project with lint
-	staticcheck ./...
-
-.PHONY: vet
-vet: ## Check the project with vet
-	go vet ./...
-
-.PHONY: fmt
-fmt: ## Run go fmt for the whole project
-	test -z $$(for d in $$(go list -f {{.Dir}} ./...); do gofmt -e -l -w $$d/*.go; done)
-
-.PHONY: imports
-imports: $(GOIMPORTS) ## Check and fix import section by import rules
-	test -z $$(for d in $$(go list -f {{.Dir}} ./...); do goimports -e -l -local $$(go list) -w $$d/*.go; done)
-
-.PHONY: cyclomatic
-cyclomatic: tools ## Check the project with gocyclo for cyclomatic complexity
-	gocyclo -over 10 `find . -type f -iname '*.go' ! -iname "*_test.go" -not -path '*/\.*'`
-
-.PHONY: static_check
-static_check: fmt imports vet lint cyclomatic ## Run static checks (fmt, lint, imports, vet, ...) all over the project
-
-.PHONY: check
-check: static_check test ## Check project with static checks and unit tests
+	golangci-lint
```

Comment:
This target is not working locally, is it possible to fix it?
```
$ make lint

golangci-lint
Smart, fast linters runner. Run it in cloud for every GitHub pull request on https://golangci.com

Usage:
  golangci-lint [flags]
  golangci-lint [command]

Available Commands:
  cache       Cache control and information
  completion  Generate the autocompletion script for the specified shell
  config      Config
  help        Help
  linters     List current linters configuration
  run         Run the linters
  version     Version

Flags:
      --color string              Use color when printing; can be 'always', 'auto', or 'never' (default "auto")
  -j, --concurrency int           Concurrency (default NumCPU) (default 16)
      --cpu-profile-path string   Path to CPU profile output file
  -h, --help                      help for golangci-lint
      --mem-profile-path string   Path to memory profile output file
      --trace-path string         Path to trace output file
  -v, --verbose                   verbose output
      --version                   Print version

Use "golangci-lint [command] --help" for more information about a command.
```

Reply:
It seems like there is a lack of `run` subcommand

Reply:
Of course, thank you.